### PR TITLE
Task 7 3 create classes button and date picker

### PR DESCRIPTION
**Pull request by vladskakun**

### CODE REVIEW

File: pages/elements/buttons.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+
+class Buttons:
```
Comment (TanyaKalistruk) [inline pages/elements/buttons.py:5]: ButtonElements

File: pages/elements/buttonElement.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+
+class ButtonElements:
```
Comment (TanyaKalistruk) [inline pages/elements/buttonElement.py:5]: ButtonElement

File: pages/elements/datepickers.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    """
+        Class for send DatePickers by css selector.
+    """
+    def __init__(self, selector):
+        self.driver = webdriver.Chrome()
+        self.css_selector = selector
+
+    def write_date_to_datepicker(self, day, month, year):
+        """
+            Method for write date on a datepicker by css selector.
```
Comment (TanyaKalistruk) [inline pages/elements/datepickers.py:16]: please describe parameters in docstring

File: pages/elements/buttons.py (hunk 1 above)
Comment (vladskakun) [reply pages/elements/buttons.py]: done

File: pages/elements/buttonElement.py (hunk 2 above)
Comment (vladskakun) [reply pages/elements/buttonElement.py]: done

File: pages/elements/datepickers.py (hunk 3 above)
Comment (vladskakun) [reply pages/elements/datepickers.py:16]: done

File: pages/landing.py [hunk 4]
```diff
@@ -4,34 +4,34 @@
...
 
     def click_join_eventexpress(self):
-        element = self.driver.find_element(By.XPATH, self.JOIN_EVENTEXPRESS_BUTTON_XPATH)
+        element = self.driver.find_element(By.CSS_SELECTOR, self.JOIN_EVENTEXPRESS_BTN_CSS)
         element.click()
 
     def click_log_out(self):
-        element = self.driver.find_element(By.XPATH, self.LOG_OUT_BUTTON_XPATH)
+        element = self.driver.find_element(By.CSS_SELECTOR, self.LOG_OUT_BTN_CSS)
         element.click()
```
Comment (TanyaKalistruk) [inline pages/landing.py:37]: We don't need these methods after implementing ButtonElement class, we can create variables for each button as instance in `__init__` method with appropriate selectors
Applicable for all classes

Comment (vladskakun) [reply pages/landing.py:37]: I had to implement it in the next PR, after push class in main branch.

Comment (vladskakun) [reply pages/landing.py]: done

File: pages/landing.py [hunk 5]
```diff
@@ -1,37 +1,22 @@
...
-        element.click()
-
-    def click_join_eventexpress(self):
-        element = self.driver.find_element(By.XPATH, self.JOIN_EVENTEXPRESS_BUTTON_XPATH)
-        element.click()
-
-    def click_log_out(self):
-        element = self.driver.find_element(By.XPATH, self.LOG_OUT_BUTTON_XPATH)
-        element.click()
+        self.sign_up = ButtonElement(self.SIGN_IN_UP_BTN_CSS)
```
Comment (TanyaKalistruk) [inline pages/landing.py:18]: self.sign_up_btn
applicable to other variables

Comment (vladskakun) [reply pages/landing.py:18]: Implemented in PR7.3

**Review by Olya27 — APPROVED** [review]

File: pages/landing.py (hunk 5 above)
Comment (vladskakun) [reply pages/landing.py:18]: done

Comment (TanyaKalistruk) [reply pages/landing.py:18]: Is not it PR7.3?)

File: pages/elements/buttonElement.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    def click_btn_by_css(self):
+        """
+            Method for click on a needed button by css selector.
+        """
+        button = self.driver.find_element(By.CSS_SELECTOR, self.selector)
+        button.click()
+
+    def click_btn_by_xpath(self):
+        """
+            Method for click on a needed button by css selector.
```
Comment (TanyaKalistruk) [inline pages/elements/buttonElement.py:22]: xpath selector

Comment (vladskakun) [reply pages/elements/buttonElement.py]: done

**Review by TanyaKalistruk — APPROVED** [review]

File: pages/elements/buttonElement.py [hunk 7]
```diff
@@ -0,0 +1,40 @@
+from telnetlib import EC
+from selenium import webdriver
+from selenium.webdriver import ActionChains
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
```
Comment (mehalyna) [inline pages/elements/buttonElement.py:5]: alphabetical order

Comment (vladskakun) [reply pages/elements/buttonElement.py]: done

**Review by mehalyna — APPROVED** [review]

**Pull request merged**